include/SongEditor.h

```diff
@@ -70,10 +70,13 @@ class SongEditor : public TrackContainerView
 	void saveSettings( QDomDocument& doc, QDomElement& element ) override;
 	void loadSettings( const QDomElement& element ) override;
 
-	ComboBoxModel *snappingModel() const;
+	ComboBoxModel* snappingModel() const;
 	float getSnapSize() const;
 	QString getSnapSizeString() const;
 
+	TimeLineWidget* timeLine;
+	PositionLine* positionLine;
```

Comment:
These do not follow the [coding conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions).

They should keep the `m_` prefix and be private. And then you could add `timeline()` and `positionLine()` getters if needed.